controllers/providers/metrics/prometheus.go

```diff
@@ -174,20 +178,36 @@ func (m *PrometheusMetrics) Unregister() {
 
 // init instantiates particular metrics
 func (m *PrometheusMetrics) init() {
+	m.metrics.K8gbEndpointHealthyRecords = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: K8gbEndpointHealthyRecords,
+			Help: "Number of healthy records observed by GSLB.",
+		},
+		[]string{"namespace", "name"},
+	)
+
 	m.metrics.K8gbGslbHealthyRecords = prometheus.NewGaugeVec(
```

Comment:
I guess the metric name should be changed too in order to reflect the updated metric description ("Number of unhealthy Services observed by GSLB.")
```suggestion
	m.metrics.K8gbGslbHealthyServices = prometheus.NewGaugeVec(
```